interfaces/apparmor/backend.go

```diff
@@ -285,6 +286,24 @@ func profileGlobs(snapName string) []string {
 	return []string{interfaces.SecurityTagGlob(snapName), nsProfile(snapName)}
 }
 
+// Determine if a profile filename is removable during core refresh/rollback.
+// On systems with unified cache directory, don't remove the snap profiles,
+// only system and snap-confine profiles. snap-confine profiles are like the
```

Comment:
What is an example for a system profile here?

Reply:
> What is an example for a system profile here?

sbin.dhclient, usr.bin.man, etc. Ie, anything either a distro package manager installs or something the admin adds.